frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -308,26 +352,44 @@ const Chart = (props: ChartProp) => {
         onRangeChange,
         propagate = true,
         onClick,
+        animationData,
     } = props;
     const dispatch = useDispatch();
     const [selected, setSelected] = useState<number[][]>([]);
-    const plotRef = useRef<HTMLDivElement>(null);
+    const plotRef = useRef<HTMLDivElement | null>(null);
+    const plotlyRef = useRef<PlotlyObject | null>(null);
     const [dataKeys, setDataKeys] = useState<string[]>([]);
+    const [frames, setFrames] = useState<{ from: Frame; to: Frame }>({
```

Comment:
Would it be a good idea to split this into 2 `useState` ?
toFrame & fromFrame ?